consent-service-impl/src/main/java/io/mosip/esignet/mapper/converter/ClaimsToStringConverter.java

```diff
@@ -0,0 +1,25 @@
+package io.mosip.esignet.mapper.converter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.mosip.esignet.api.dto.Claims;
+import io.mosip.esignet.core.exception.EsignetException;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.Converter;
+import org.modelmapper.spi.MappingContext;
+
+import static io.mosip.esignet.core.constants.ErrorConstants.INVALID_CLAIM;
+
+@Slf4j
+public class ClaimsToStringConverter implements Converter<Claims, String> {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Instead of declaring ObjectMapper in all the converter classes, Can we declare it in ConsentMapper and then pass it as a constructor argument of the converter class?

Reply:
done